data.sql

```diff
@@ -4,3 +4,11 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Gabumon', '2018-11-15', 2, TRUE, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Pikachu', '2021-01-07', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES('Devimon', '2017-05-12', 5, TRUE, 11);
+ALTER TABLE animals ADD COLUMN species VARCHAR(50);
```

Comment:
- [ ] Your work on the tables is impressive :+1: :100:. However, it is important to note that any action that creates or modifies the table structure, including ALTER TABLE, should be recorded in the `schema.sql` file as per project requirement.